CHANGELOG.md

```diff
@@ -21,6 +21,7 @@
 - Minor: Added ability to execute commands on chat messages using the message context menu. (#3738, #3765)
 - Minor: Added `/copy` command. Usage: `/copy <text>`. Copies provided text to clipboard - can be useful with custom commands. (#3763)
 - Minor: Removed total views from the usercard, as Twitch no longer updates the number. (#3792)
+- Minor: Speed up program loading time GO FAST (#3813)
```

Comment:
As per this changelog entry I don't think it's even needed at all. It actually sounds wrong because the whole speedup here is just placebo imo. Nothing speeds up, you just perform certain actions a bit sooner - not faster. Technically program won't finish "loading" until you have JOINed all the channels, which is sadly limitted by twitch to ~20 JOINs/10s so you'll have to wait regardless.

And in the case if I'm actually wrong and the changelog entry is to be kept, the last two words in caps don't make any sense, remove them and end the sentence with a comma (that suggestion is only if we decide to keep this, but my preference is to completely remove it).